apps/src/sites/studio/pages/levels/editors/fields/_validation_code.js

```diff
@@ -1,5 +1,9 @@
 import $ from 'jquery';
 import initializeCodeMirror from '@cdo/apps/code-studio/initializeCodeMirror';
+import getScriptData from '@cdo/apps/util/getScriptData';
+
+const data = getScriptData('map');
```

Comment:
I would suggest renaming `data` to descriptive enough to be self-documenting (e.g. `sampleValidationFunctionsMap`). Or, if a good name feels too long or not descriptive enough, a short comment clarifying that this is a map containing the text sample validation functions would be fine instead.

Reply:
Thanks again @davidsbailey! I performed the request clean up. Let me know if you see anything else that could be improved!